### PR TITLE
Add Error component

### PR DESCRIPTION
**Pull request by tnlitv**
#### Please tick a box ###
- [x] **feature** _(A new feature_)

#### If you're adding a feature, is it documented in a storybook story?
- [x] Yes

#### Are the components you're working on reusable between brands?
- [x] Yes _(Using variables that are defined in the default theme)_

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/Error.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+import propTypes from 'prop-types';
+import { getColor, getVariable } from '../utils';
+import { Col, XLargePlugHeading, SmallHeading } from '..';
+
+const ErrorWrapper = Col.extend`
+	text-align: center;
+	font-family: Cabin,sans-serif;
+    padding: 3rem;
```
[the remark this answers is not in the document]
Comment (tnlitv) [reply src/atoms/Error.js]: Thanks, that's a life-saver! I wonder though if we can make linter notice such things

File: stories/typography/error.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import { Error } from '../../src/atoms/Error';
+
+export default () => (
+	<section>
+		<HugeHeading>Error message</HugeHeading>
+		<p>Optional attributes can be specified: detailed message and callback on Retry button click</p>
+
+		<Heading>Default</Heading>
+		<DemoContainer>
+			<Error />
```
[the remark this answers is not in the document]
Comment (tnlitv) [reply stories/typography/error.js]: But I show usages below. Or am I misunderstanding something?

File: src/atoms/Error.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	color: ${getColor('primary')};
+	font-size: 7rem;
+	line-height: 7rem;
+`;
+
+const Message = SmallHeading.extend`
+	font-weight: 400;
+`;
+
+const ReloadButton = styled.button`
```
[the remark this answers is not in the document]
Comment (tnlitv) [reply src/atoms/Error.js]: Good point, I just didn't see Button in storybook and thought we don't have one
Gonna fix it now (and add it to storybook)

File: stories/typography/error.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (SindreSvendby) [reply stories/typography/error.js]: my bad!

**Pull request merged**